src/views/ApartmentApp/components/StudentApplication/components/ApartmentHeader/index.js

```diff
@@ -8,68 +8,83 @@ const ApartmentHeader = ({
+        <Typography variant="body1">
+          You are on an existing application for this semester, but you are not assigned as the
+          editor.
+          <br />
+          Only the application editor may edit the application.
+        </Typography>
+      </Grid>
+      <Grid item xs={6} sm={4}>
+        <Button
+          variant="contained"
+          onClick={onShowApplication}
+          color="primary"
+          fullWidth
+          disabled={applicationCardsOpen}
+        >
+          View your application
+        </Button>
+      </Grid>
+    </React.Fragment>
+  );
+
   return (
     <Card>
       <CardContent>
         <Grid container direction="row" justify="flex-end" spacing={2}>
-          {!applicationID ? (
-            <React.Fragment>
-              <Grid item xs={6} sm={8}>
-                <Typography variant="body1">
-                  Placeholder Text
-                  <br />
-                  No existing applications found
-                </Typography>
-              </Grid>
-              <Grid item xs={6} sm={4}>
-                <Button
-                  variant="contained"
-                  onClick={onShowApplication}
-                  color="primary"
-                  fullWidth
-                  disabled={applicationCardsOpen}
-                >
-                  Create a new application
-                </Button>
-              </Grid>
-            </React.Fragment>
-          ) : userProfile.AD_Username === editorUsername ? (
-            <React.Fragment>
-              <Grid item xs={6} sm={8}>
-                <Typography variant="body1">Existing application for this semester:</Typography>
-              </Grid>
-              <Grid item xs={6} sm={4}>
-                <Button
-                  variant="contained"
-                  onClick={onShowApplication}
-                  color="primary"
-                  fullWidth
-                  disabled={applicationCardsOpen}
-                >
-                  Edit your application
-                </Button>
-              </Grid>
-            </React.Fragment>
-          ) : (
-            <React.Fragment>
-              <Grid item xs={6} sm={8}>
-                <Typography variant="body1">
-                  Only the application editor may edit the application.
-                </Typography>
-              </Grid>
-              <Grid item xs={6} sm={4}>
-                <Button
-                  variant="contained"
-                  onClick={onShowApplication}
-                  color="primary"
-                  fullWidth
-                  disabled={applicationCardsOpen}
-                >
-                  View your application
-                </Button>
-              </Grid>
-            </React.Fragment>
-          )}
+          {!applicationID
+            ? newApplicationContent
+            : userProfile.AD_Username === editorUsername
+            ? editApplicationContent
+            : viewApplicationContent}
```

Comment:
All three of these variables are literally the same, except for a different `<Typography>` content and button label. I think this component would end up much smaller and much more readable if you made those two pieces into variables instead.
```suggestion
          <Grid item xs={6} sm={8}>
            <Typography variant="body1">{text}</Typography>
          </Grid>
          <Grid item xs={6} sm={4}>
            <Button
              variant="contained"
              onClick={onShowApplication}
              color="primary"
              fullWidth
              disabled={applicationCardsOpen}
            >
              {buttonLabel}
            </Button>
          </Grid>
```

Then you could replace all the logic above the return with simply:
```
let text, buttonLabel;

  if (!applicationID) {
    text = (
      <>
        You are not on any applications for the current semester.
        <br />
        Click the button to begin a new application.
      </>
    );
    buttonLabel = 'Create a new application';
  } else if (userProfile.AD_Username === editorUsername) {
    text = 'You are on an existing application for this semester.';
    buttonLabel = 'Edit your application';
  } else {
    text = (
      <>
        You are on an existing application for this semester, but you are not assigned as the
        editor.
        <br />
        Only the application editor may edit the application.
      </>
    );
    buttonLabel = 'View your application';
  }
```

Reply:
Oh that's a really good point, that was annoying me trying to figure out how to simplify that when there was so much similar between them

Reply:
Also, I never knew that syntax was a thing:
```
<>
  stuff
</>
```

Reply:
I just pushed an change based on this

Reply:
Yes, `<>SomeJSX</>` is shorthand for `<React.Fragment>SomeJSX</React.Fragment>`. Being shorthand, it's less obvious/explicit, but I think it's at least as readable and still conveys the intent of wrapping the elements together but not providing any wrapping functionality.